maven-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.ozonehis</groupId>
+    <artifactId>ozone-distro-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Any way to manage this version of the parent to the current version in `../maven-parent/` ?

Reply:
Not as directly as I'd like. What I can do (pretty easily) is add a step to the post-generate script that will set this version to match the version of the maven-archetype itself...